spec/pluralrules.html

```diff
@@ -46,20 +46,18 @@
     </emu-clause>
 
     <emu-clause id="sec-GetOperands" aoid="GetOperands">
-       <h1>GetOperands (x)</h1>
+       <h1>GetOperands (s)</h1>
        <p>
-         When the GetOperands abstract operation is called with argument _n_ (which must be a finite Number value), it returns an Object value with the list of plural operands used by plural rules functions.
+         When the GetOperands abstract operation is called with argument _s_ (which must be a String matching the regular expression pattern `-?\d*(\.\d*)?`, it returns an Object value with the list of plural operands used by plural rules functions.
```

Comment:
See thread on line 158 for rationale for string input here. Regarding the regexp: should it be replaced by a wordier explanation for the input restrictions?